### ISSUE TITLE
Can't login with newly created user

### ISSUE THREAD

**Issue by hynese** [mail-in-a-box/mailinabox#1626]
I'm having an issue logging in with a newly created user. The new user is created via the /admin panel.

I notice that if I tail the logs before clicking "Login" and then click "Login":

`sudo tail -f /var/log/mail.log`

, I see a stream of lines quickly filling up the logs...

```
Aug 12 00:28:49 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5412, secured, session=<KGEMxd+P5JR/AAAB>
Aug 12 00:28:49 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5414, secured, session=<vZsNxd+P6pR/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5416, secured, session=<Sw8Pxd+P8JR/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5418, secured, session=<RUkQxd+P9pR/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5420, secured, session=<j4YRxd+P/JR/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5422, secured, session=<9r4Sxd+PApV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5424, secured, session=<8PcTxd+PCJV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5426, secured, session=<kjkVxd+PDpV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5428, secured, session=<Q3kWxd+PFJV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5430, secured, session=<wbEXxd+PGpV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5432, secured, session=<C+8Yxd+PIJV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5434, secured, session=<Rigaxd+PJpV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:50 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5436, secured, session=<TVwbxd+PLJV/AAAB>
Aug 12 00:28:50 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5438, secured, session=<6Y8cxd+PMpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5440, secured, session=<CA4exd+POJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5442, secured, session=<O0Efxd+PPpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5444, secured, session=<o3Ugxd+PRJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5446, secured, session=<qqshxd+PSpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5448, secured, session=<m+sixd+PUJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5450, secured, session=<UyAkxd+PVpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5452, secured, session=<SFklxd+PXJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5454, secured, session=<uYomxd+PYpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5456, secured, session=<bcUnxd+PaJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5458, secured, session=<Vfgoxd+PbpV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412
Aug 12 00:28:51 imap-login: Info: Login: user=<test@mydomain.org>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=5460, secured, session=<9CUqxd+PdJV/AAAB>
Aug 12 00:28:51 imap(test@mydomain.org): Info: Logged out in=12 out=412

If I do:

`sudo systemctl stop dovecot.service`

My new user logs in... 😄 

I then just do:

`sudo systemctl start dovecot.service`

And everything is ok again...

It's as if there's some timing race condition? Because the new user meta info returns so quickly, there's some other check that assumes the first call takes a bit longer?

I'm not sure. This has been bugging me for a while... Thought I should report. 

I upgraded to 0.42b tonight. No issues with the upgrade. However, this login issue for new users still persists. Thanks ya'll.

**Comment by jl2035** [comment]
+1 New install, unable to login

**Comment by cezarmathe** [comment]
+1, my box is 4 months old, after the last upgrade I can't log into my mail account.

**Comment by jl2035** [comment]
I moved to iRedMail, still struggling with configuration.. emails ending in Spam etc.. This is really hard to setup.

**Comment by JoshData** [comment]
@hynese @jl2035 What service are you trying to log in to? IMAP? Is there an error message? Are your passwords plain US English letters?

**Comment by jl2035** [comment]
Consider me out of this issue, but it happened to me also.. Fresh install, unable to login to admin.

**Comment by JoshData** [comment]
Sorry meant to tag @cezarmathe!

**Comment by cezarmathe** [comment]
yup, imap, tried on thunderbird on arch and on the default mail client on windows. yes, my password is in plain english.

**Comment by cezarmathe** [comment]
> Consider me out of this issue, but it happened to me also.. Fresh install, unable to login to admin.

it's actually really easy to set up, you might have just tried setting up at a bad time lol. when i set up my box it tool me about an hour or so with dns not managed by mailinabox.
